### PR TITLE
test: e2e test for zeebe userTask completion

### PR DESCRIPTION
**Pull request by esraagamal6**
Add an e2e test for completing a zeebe userTask

## Related issues

closes #

### CODE REVIEW

File: tasklist/config/docker-compose.identity.yml
Comment (vsgoulart) [inline tasklist/config/docker-compose.identity.yml:1]: @esraagamal6 Let's revert the changes in all the Yaml files

File: tasklist/client/e2e/tests/task-details.spec.ts [hunk 1]
```diff
@@ -156,6 +158,58 @@ test.describe('task details page', () => {
...
+    formJSDetailsPage,
+  }) => {
+    await taskPanelPage.openTask('Zeebe_user_task');
+    await taskDetailsPage.unassignButton.click();
+    await expect(formJSDetailsPage.completeTaskButton).toBeDisabled({
+      timeout: 60000,
+    });
+    await taskDetailsPage.assignToMeButton.click();
+    await expect(formJSDetailsPage.completeTaskButton).toBeEnabled({
+      timeout: 60000,
```
Comment (vsgoulart) [inline tasklist/client/e2e/tests/task-details.spec.ts:174]: We don't have these timeouts in any of the other tests, why is this necessary here?

**Review by vsgoulart — COMMENTED** [review]
@esraagamal6 I think we should actually just migrate all existing tests to deployed user tasks tasks and keep a test for the old method, since the deployed tasks will be the only thing we'll have in the future

File: tasklist/config/docker-compose.identity.yml
Comment (esraagamal6) [reply tasklist/config/docker-compose.identity.yml:1]: It was pushed by mistake

File: .github/workflows/tasklist-e2e-tests.yml
Comment (houssain-barouni) [inline .github/workflows/tasklist-e2e-tests.yml:1]: tasklist is now using zeebe rest api v2,
you need to change `camunda/zeebe:8.5.3` to `camunda/zeebe:SNAPSHOT`

Comment (vsgoulart) [reply .github/workflows/tasklist-e2e-tests.yml:1]: @esraagamal6 :point_up:

**Review by vsgoulart — COMMENTED** [review]
@esraagamal6 I did a big refactor on the page objects some weeks ago, I think there are some rebase mistakes
We also have lots of linting errors, you can check them locally by running `yarn run lint`

**Review by vsgoulart — APPROVED** [review]

File: .github/workflows/tasklist-e2e-tests.yml [hunk 2]
```diff
@@ -3,27 +3,27 @@ name: Tasklist E2E Tests
 on:
   push:
     branches:
-      - 'main'
```
Comment (houssain-barouni) [inline .github/workflows/tasklist-e2e-tests.yml:6]: is the change from single to double quotes intentional ? or made by a formatting linter ?

Comment (esraagamal6) [reply .github/workflows/tasklist-e2e-tests.yml:6]: @houssain-barouni made by linter
